Sources/PT.PM.Matching/Patterns/PatternReturn.cs

```diff
@@ -0,0 +1,48 @@
+﻿using PT.PM.Common;
+using PT.PM.Common.Nodes;
+using PT.PM.Common.Nodes.Statements;
+using System;
+
+namespace PT.PM.Matching.Patterns
+{
+    public class PatternReturn : PatternUst
+    {
+        public PatternUst Pattern { get; set; } = PatternAny.Instance;
```

Comment:
As I understand `ReturnExpression` or just `Expression` is a more appropriate name instead of `Pattern`.

`Pattern` name used for artificial logical nodes (NOT).